cmd/argocd/commands/project.go

```diff
@@ -322,7 +322,7 @@ func NewProjectAddDestinationCommand(clientOpts *argocdclient.ClientOptions) *co
 // NewProjectRemoveDestinationCommand returns a new instance of an `argocd proj remove-destination` command
 func NewProjectRemoveDestinationCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var command = &cobra.Command{
-		Use:   "remove-destination PROJECT SERVER NAMESPACE",
+		Use:   "remove-destination PROJECT SERVER NAMESPACE SERVICEACCOUNT",
```

Comment:
This change is not required. As we have a separate object DestinationServiceAccount under AppProject